src/include/miopen/conv/problem_description.hpp

```diff
@@ -29,7 +29,9 @@
 #include <miopen/conv_algo_name.hpp>
 #include <miopen/convolution.hpp>
 #include <miopen/names.hpp>
+#if MIOPEN_ENABLE_SQLITE
```

Comment:
I believe we can move this into sqlite_db.hpp

Reply:
Fixed